sample_data/submitSamples.json

```diff
@@ -0,0 +1,140 @@
+{
+  "study": {
+    "key": 1,
+    "program": "dev",
```

Comment:
we still hard code program/project name here?

Reply:
No, that is updated in the other pull request, branch feat/graphqlQuery